xgcm/grid.py

```diff
@@ -134,6 +144,13 @@ def __init__(
             # fall back on comodo conventions
             self.coords = comodo.get_axis_positions_and_coords(ds, axis_name)
 
+        if direction=="increasing" or direction is None:
+            self.direction_sign=1
+        elif direction=="decreasing":
+            self.direction_sign=-1
+        else:
+            raise ValueError(f"Axis direction not recognized. Expecting `increasing` or `decreasing` or None, got `{direction}`.")
```

Comment:
Similar to my above comment, using a boolean value here would obviate a lot of this logic.